index.bs

```diff
@@ -318,6 +339,285 @@ NOTE: The size of the last layer can be determined by subtracting the sum of the
 
 NOTE: [[!AV1]] supports encoding either 3-component images (whose semantics are given by the <code>matrix_coefficients</code> element), or 1-component images (monochrome). When an image requires a different number of components, multiple auxiliary images may be used, each providing additional component(s), according to the semantics of their <code>aux_type</code> field. In such case, the maximum number of components is restricted by number of possible items in a file, coded on 16 or 32 bits.
 
+<h3 id="derived-images">Derived Image Items and Sequences</h3>
+
+<h4 id="sample-transform">Sample Transform Derived Image Item</h4>
+
+In these sections, a "sample" refers to the value of a pixel for a given channel.
+
+<h5 id="sample-transform-definition">Definition</h5>
+
+When a [=MIAF derived image item=] is of type <dfn value export for="Sample Transform Derived Image Item Type">sato</dfn>, it is called a <dfn export>Sample Transform Derived Image Item</dfn>, and its reconstructed image is formed from a set of input image items, constants and operators.
+
+The input images are specified in, and in the same order as the <code>[=SingleItemTypeReferenceBox=]</code> or <code>[=SingleItemTypeReferenceBoxLarge=]</code> entries of type 'dimg' for this [=Sample Transform Derived Image Item=] within the <code>[=ItemReferenceBox=]</code>. In the <code>[=SingleItemTypeReferenceBox=]</code> or <code>[=SingleItemTypeReferenceBoxLarge=]</code> of type 'dimg', the value of the <code>[=from_item_ID=]</code> field identifies the [=Sample Transform Derived Image Item=], and the values of the <code>[=to_item_ID=]</code> field identify the input images. There are <code>[=reference_count=]</code> input image items as specified by the <code>[=ItemReferenceBox=]</code>.
```

Comment:
Nonconstructive criticism: The first sentence of this paragraph is long and complicated. It is very hard to understand. I think it would help if we could break it into two simpler sentences.

Reply:
I am open to suggestions.

Reply:
There are two reasons why the first sentence is hard to understand. The first is that it describes a complex set of boxes. Since I am not familiar with that set of boxes, I am not qualified to suggest changes. The second reason is the "are specified in, and in the same order as" part. It crams two notions into the sentence. Although it shortens the sentence, it makes the sentence even more complex.

We can address the second problem by splitting the first sentence into two sentences. One describes the complex set of boxes, and the other describes the order of the input images.